bagisto/bagisto
# Fixed 8636 - Admin panel date picker style in dark mode

## Issue Reference
#8636 

## Description
Hi,
In this PR, I fixed the issue of date pickers in the admin panel by bringing dark CSS styles from the flatpickr dark mode CSS file.

Note: Since `postcss-nested` is not configured for tailwind, there's a warning about that. I can add it to the tailwind config, but first, wanna know if the main changes are ok.

## How To Test This?
1. Login to the admin panel
2. Use dark mode
3. Check out any date picker
